docs/self-managed/components/management-identity/configuration/connect-to-an-oidc-provider.md

```diff
@@ -51,7 +51,7 @@ configuration](#component-specific-configuration) to ensure the components are c
      - **Optimize**
      - **Management Identity**
      - **Web Modeler API**
-   - Web applications requiring confidential access/a confidential client:
+   - Web applications requiring public access/a public client:
```

Comment:
I assume this was changed unintentionally when the guide was migrated to Management Identity for 8.8.

In 8.7, the application types are correctly documented: https://docs.camunda.io/docs/self-managed/identity/configuration/connect-to-an-oidc-provider/#configuration